vulnerabilities/phantom-functions.md

```diff
@@ -0,0 +1,134 @@
+# Phantom Functions
+
+A phantom function refers to a function that is non-existent in a smart contract but is accepted without reverting when called. This means that the function call does not match any existing function name in the contract but is caught by the fallback function.
+
+Invoking a non-existent function within a smart contract typically results in the transaction reverting, but luckily there exists a special function that is executed when a function that does not exist on the contract is called: the `fallback()` function.
+
+While this special function is beneficial for handling unexpected calls, it can also introduce significant vulnerabilities if not properly implemented and validated.
```

Comment:
I think not fair to say that the fallback function is introducing vulnerabilities, but rather that not carefully integrating with a contract that has a fallback function can introduce vulnerabilities

Reply:
Doesn't the entirety of sentence indicate that, i.e "...While this special function is beneficial for handling unexpected calls, it can also introduce significant vulnerabilities if not properly implemented and validated"